### ISSUE TITLE
Padding UI can persist on save

### ISSUE THREAD

**Issue by ajlende** [WordPress/gutenberg#31839]
## Description

The padding/margin UI is being stored in the block attributes under `style.visualizers`, so if you save while hovered over the padding input, the UI persists upon reloading.

```html
<!-- wp:cover {"url":"https://placekitten.com/500/500","id":1,"align":"center","style":{"spacing":{"padding":{"top":"128px","right":"128px","bottom":"128px","left":"128px"}},"visualizers":{"padding":{"top":true,"right":true,"bottom":true,"left":true}}}} -->
<div class="wp-block-cover aligncenter has-background-dim" style="padding-top:128px;padding-right:128px;padding-bottom:128px;padding-left:128px">
  <img class="wp-block-cover__image-background wp-image-1" alt="" src="https://placekitten.com/500/500" data-object-fit="cover"/>
  <div class="wp-block-cover__inner-container">
    <!-- wp:spacer -->
    <div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
    <!-- /wp:spacer -->
  </div>
</div>
<!-- /wp:cover -->
```

## Step-by-step reproduction instructions

1. Create a cover block
2. Add some padding
3. While hovering on the padding input, save the document with the save shortcut, command+s/control+s
4. Reload the page

## Screen recording

![padding-ui-persist-on-save](https://user-images.githubusercontent.com/5129775/118281951-773d3d80-b469-11eb-821d-918d257a7b63.gif)

**Comment by Mamaduka** [comment]
Hi, @ajlende

Do you think we can close this one now that #40505 is merged?

**Comment by ndiego** [comment]
I have tested, and this is no longer an issue in the latest version of Gutenberg. I believe this can be closed.

**Comment by t-hamano** [comment]
I tested again, and it didn't reproduce in the latest Gutenberg.

I would like to close this issue as it should have been resolved by #40505.
(The PR was merged in 13.2, so it’s not include in WP 6.0)

Here are the test results:

## WordPress 6.0.1 without Gutenberg (Reproduced)

https://user-images.githubusercontent.com/54422211/179897061-f9f1637f-3259-4052-bc6f-50e5eeaa0b40.mp4

## WordPress 6.0.1 with Gutenberg 13.7.0 (Not reproduced)

https://user-images.githubusercontent.com/54422211/179897094-f83966a6-d92f-4a93-94d2-9f6a4b291f81.mp4

The visualizer was limited to a display time, so I tried to save while moving the mouse up/down, but there was no problem.

https://user-images.githubusercontent.com/54422211/179897793-e7e1fe92-8a8f-489c-ae87-e91ecd09ea15.mp4
